chcemvediet/apps/inforequests/admin.py

```diff
@@ -394,18 +410,62 @@ def delete_constraints(self, obj):
                 u'{} is the only action in the branch.'.format(admin_obj_format(obj))))
         return constraints
 
+    def can_snooze(self, obj):
+        if (obj.type in [Action.TYPES.EXPIRATION, Action.TYPES.APPEAL_EXPIRATION]
+                and obj.previous_action):
+            return True
+        return False
```

Comment:
Teraz ked tie nie je natlacene v jednom velkom `if` ale je to separatna metoda, tak ten `if` mozes rozdelit, aby to bolo citatelnejsie. Nieco ako:
```
if ...:
  return False
if ...:
  return False
return True
```

Reply:
:ok: